assets/src/edit-story/components/tabview/index.js

```diff
@@ -53,51 +57,66 @@ const Tab = styled.li.attrs(({ isActive }) => ({
   role: 'tab',
   'aria-selected': isActive,
 }))`
-  text-align: center;
   cursor: pointer;
   border: none;
   background: none;
-  color: ${({ theme }) => theme.colors.fg.tertiary};
-  font-family: ${({ theme }) => theme.DEPRECATED_THEME.fonts.tab.family};
-  font-size: ${({ theme }) => theme.DEPRECATED_THEME.fonts.tab.size};
-  font-weight: ${({ theme }) => theme.DEPRECATED_THEME.fonts.tab.weight};
-  padding: 10px 0px;
-  margin: 0 20px;
-  margin-bottom: -1px;
+  padding: 0 4px;
+  margin: 10px 12px 9px;
+  height: 32px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   position: relative;
+  transition: color 0.2s ease;
+  color: ${({ theme, isActive }) =>
+    isActive ? theme.colors.fg.primary : theme.colors.fg.tertiary};
+
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+  ${({ theme }) =>
+    themeHelpers.focusableOutlineCSS(
+      theme.colors.border.focus,
+      theme.colors.bg.secondary
+    )};
+
+  :hover {
+    color: ${({ theme }) => theme.colors.fg.primary};
+  }
 
   ${({ isActive, theme }) =>
     isActive &&
-    `
-    padding-bottom: 8px;
-    border-bottom: 2px solid ${theme.colors.fg.primary};
-  `}
+    css`
+      ::after {
+        content: '';
+        position: absolute;
+        background-color: ${theme.colors.border.selection};
+        height: 2px;
+        border-radius: 1px;
+        bottom: -9px;
```

Comment:
Looks like this should be `-10px` instead.
9:
<img width="241" alt="Screenshot 2021-03-03 at 12 34 27" src="https://user-images.githubusercontent.com/3294597/109793053-75a6fc00-7c14-11eb-81b8-51db20f10edf.png">
10:
<img width="242" alt="Screenshot 2021-03-03 at 12 34 20" src="https://user-images.githubusercontent.com/3294597/109793058-76d82900-7c14-11eb-8bd1-e1e091e7972b.png">

Reply:
Thanks, well spotted 🎉 . Fixed in dce7451c5.